tests/python/unittest/test_schedule_schedule_ops.py

```diff
@@ -248,10 +248,26 @@ def test_schedule_cache_relayout3():
     bounds = tvm.schedule.InferBound(s)
     stmt = tvm.schedule.ScheduleOps(s, bounds)
 
+def test_schedule_cache_relayout4():
+    def _compute(*indice):
+        return A(*indice) + 1, B(*indice) / 2
+    m = tvm.var('m')
+    n = tvm.var('n')
+    A = tvm.placeholder((m*4, n), name='A')
+    B = tvm.placeholder((m*4, n), name='B')
```

Comment:
please add an end to end test that compiles and runs a program with cache_write multiple schedule. possibly in tests/integration/test_ewise.py